flake.nix

```diff
@@ -79,33 +91,23 @@
 
             doCheck = true;
 
-            cargoTestCommands = x: x ++ [
-              # clippy
-              ''cargo clippy --all --all-features --tests -- -D clippy::pedantic -D warnings''
-              # rustfmt
-              ''cargo fmt -- --check''
-            ];
+            # for some tests
+            requiredSystemFeatures = lib.optionals (!pkgs.stdenv.isDarwin && doCheck) [ "recursive-nix" ];
 
-            overrideMain = _: {
-              postInstall = ''
-                set -euo pipefail
-
-                # Provide a symlink from `agenix` to `ragenix` for compat
-                ln -sr "$out/bin/ragenix" "$out/bin/agenix"
-
-                # Install shell completions
-                res=$(
-                  cat "$cargo_build_output_json" \
-                    | jq 'select(.reason == "build-script-executed") | select(.env != []) | select(.env[0][0] | startswith("RAGENIX"))' \
-                    | jq '.env | map({(.[0]):(.[1])}) | add'
-                )
-
-                set +u # required due to `installShellCompletion`'s implementation
-                installShellCompletion --bash "$(echo "$res" | jq -r '.RAGENIX_COMPLETIONS_BASH')"
-                installShellCompletion --zsh  "$(echo "$res" | jq -r '.RAGENIX_COMPLETIONS_ZSH')"
-                installShellCompletion --fish "$(echo "$res" | jq -r '.RAGENIX_COMPLETIONS_FISH')"
-              '';
-            };
+            postInstall = ''
+              set -euo pipefail
+
+              # Provide a symlink from `agenix` to `ragenix` for compat
+              ln -sr "$out/bin/ragenix" "$out/bin/agenix"
+
+              # Stdout of build.rs
+              buildOut=$(find "$tmpDir/build" -type f -regex ".*\/ragenix-[a-z0-9]+\/output")
```

Comment:
`cargo_build_output_json` was a naersk specific interface.

Reply:
It would be great if we could still rely on the explicit paths we output here:

https://github.com/yaxitech/ragenix/blob/1dcf1ba122f06392737661d1f5f19e01da9cef3f/build.rs#L19-L35

If that's not possible, please remove the code in `build.rs`.

Reply:
On cel direct commenting didn't work for some reason. Now on PC:

Yes we still do that a few lines below that line. That line only captures the stdout output file (called just: output) from the build folder.

https://github.com/yaxitech/ragenix/pull/62/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R115-R117